services/localizely/localizely.service.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService, InvalidResponse } = require('..')
+const { coveragePercentage } = require('../color-formatters')
+
+const keywords = [
+  'l10n',
+  'i18n',
+  'localization',
+  'internationalization',
+  'translation',
+  'translations',
+]
+
+const documentation = `
+  <p>
+    The read-only API token from the Localizely account is required to fetch necessary data.
+    <br/>
+    You can find more details regarding API tokens under <a href="https://app.localizely.com/account" target="_blank">My profile</a> page.
+  </p>
+  `
+
+const schema = Joi.object({
+  strings: Joi.number().required(),
+  reviewedProgress: Joi.number().required(),
+  languages: Joi.array()
+    .items(
+      Joi.object({
+        langCode: Joi.string().required(),
+        langName: Joi.string().required(),
+        strings: Joi.number().required(),
+        reviewed: Joi.number().required(),
+        reviewedProgress: Joi.number().required(),
+      })
+    )
+    .required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  token: Joi.string().required(),
+  languageCode: Joi.string().regex(/^[a-z]{2}(-[A-Z][a-z]{3})?(-[A-Z]{2})?$/),
+}).required()
+
+module.exports = class Localizely extends BaseJsonService {
+  static category = 'other'
+
+  static route = {
+    base: 'localizely/progress',
+    pattern: ':projectId/:branch*',
+    queryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'Localizely overall progress',
+      keywords,
+      documentation,
+      namedParams: {
+        projectId: '1349592f-8d05-4317-9f46-bddc5def11fe',
```

Comment:
This is currently returning a not authorized message in the review app. Not required but we generally like to have the example badges use values that'll work when users plug them in. 

What about using a different project?

```suggestion
        projectId: '5cc34208-0418-40b1-8353-acc70c95f802',
```

https://shields-staging-pr-5472.herokuapp.com/localizely/progress/1349592f-8d05-4317-9f46-bddc5def11fe?token=0f4d5e31a44f48dcbab966c52cfb0a67c5f1982186c14b85ab389a031dbc225a
![](https://shields-staging-pr-5472.herokuapp.com/localizely/progress/1349592f-8d05-4317-9f46-bddc5def11fe?token=0f4d5e31a44f48dcbab966c52cfb0a67c5f1982186c14b85ab389a031dbc225a)

vs.

https://shields-staging-pr-5472.herokuapp.com/localizely/progress/5cc34208-0418-40b1-8353-acc70c95f802?token=0f4d5e31a44f48dcbab966c52cfb0a67c5f1982186c14b85ab389a031dbc225a
![](https://shields-staging-pr-5472.herokuapp.com/localizely/progress/5cc34208-0418-40b1-8353-acc70c95f802?token=0f4d5e31a44f48dcbab966c52cfb0a67c5f1982186c14b85ab389a031dbc225a)

Reply:
Examples updated with real values within [a2803c](https://github.com/badges/shields/pull/5472/commits/a2803ccbce8465fb866db905d9e957b892b7caf6) commit.